src/Components/SingleArticle.jsx

```diff
@@ -0,0 +1,36 @@
+import { useParams } from "react-router-dom";
+import { getArticleById } from "./api";
+import { useState, useEffect } from "react";
+export default function SingleArticle() {
+	const { article_id } = useParams();
+	const [singleArticle, setSingleArticle] = useState({});
```

Comment:
we need a loading state here to let the user know that the webpage is on its way and there isnt an error